contracts/cw1155-base/helpers.ts

```diff
@@ -0,0 +1,393 @@
+import axios from  "axios";
```

Comment:
Get the feeling that this file won't currently work, plus there's a bunch of stuff that only seems to relate to CW721. Has it been tested at all? Maybe worth binning if it doesn't work.